pkg/consolegraphql/resolvers/resolver_address.go

```diff
@@ -187,5 +187,15 @@ func (r *mutationResolver) PurgeAddress(ctx context.Context, input metav1.Object
 }
 
 func (r *queryResolver) AddressCommand(ctx context.Context, input v1beta1.Address) (string, error) {
-	return "oc apply blah", nil
+
+	return `apiVersion: enmasse.io/v1beta1
+kind: Address
```

Comment:
what is the reason to not use yaml serialisation of the object?

Reply:
@k-wall I will provide a sample.  It was giving me a number of empty fields, that made it look more complicated.  I'll see if there is a way to remove them.

Reply:
@k-wall  I haven't been able to find a way to get the yaml marshaller to remove the k8s.io.ObjectMeta objects empty field.  If I could tell it to look at the json's tags (instead of yaml's tags), it should work. 
```kind: AddressSpace
  apiversion: enmasse.io/v1beta1
  metadata:
    name: mynamespace
    generatename: ""
    namespace: myaddressspace
    selflink: ""
    uid: 7ba0337f-9037-42e6-b00a-2831cf51a77e
    resourceversion: ""
    generation: 0
    creationtimestamp: "0001-01-01T00:00:00Z"
    deletiontimestamp: null
    deletiongraceperiodseconds: null
    labels: {}
    annotations: {}
    ownerreferences: []
    initializers: null
    finalizers: []
    clustername: ""
    managedfields: []
  spec:
    plan: standard
    type: standard-unlimited
    authenticationservice:
      name: none-authentication```

Reply:
There might be a more elegant solution, but I think the following process would produec yaml with the field names following the JSON annotations:

1) Marshall Object to JSON (byte array)
2) Unmarshal the byte array to map[interface{}]interface{}
3) Marshall the map to yaml

Reply:
I have swtiched to use JSON -> MAP -> YAML marshalling.  Let me know, if you think it is better this way or before.